frontend/javascripts/admin/statistic/statistic_view.js

```diff
@@ -90,7 +90,7 @@ class StatisticView extends React.PureComponent<{}, State> {
 
   selectDataPoint = ({ chartWrapper }: GoogleCharts) => {
     const chart = chartWrapper.getChart();
-    const indicies = chart.getSelection().selection[0];
+    const indicies = chart.getSelection()[0];
```

Comment:
Could you test this in multiple browsers? I'm wondering whether the library exposes an inconsistent behavior somewhere. Since 2019-02-14 we use `"react-google-charts": "^2.0.0"` and on 2020-06-23, @MichaelBuessemeyer changed this exact line (see https://github.com/scalableminds/webknossos/commit/fe93f6615fa4cf14971a1050170c8210be9e24eb#diff-823e1a6f8f8ef290cfeec2ab2219cc92dea4da813f1c1717b833b70346953b75R93). Now your change would basically revert his change :smile: I assume you both had good reasons to change the code, but they are contradicting each other. I'm afraid that we might get another bug report soon. So, it would be good to find out what's going on. Testing multiple browsers might be a good place to start :thinking: 

Maybe it also depends on where one clicks exactly or something like that?

@MichaelBuessemeyer Do you know more about this?

Reply:
I just talked with Gritta about this change / bug and it looks like her versions is correct. Maybe I messed this up last year :sweat_smile: :swan:? 

We checked the fix in firefox and chromium. It would be nice if oucld also check the fix in the browsers you have installed @philippotto :robot: